### PR TITLE
Update nvdaHelper readme

### PR DESCRIPTION
**Pull request by feerrenrut**
### Link to issue number:
None

### Summary of the issue:
The instructions for creating a Visual Studio project for the nvdaHelper C++ code were either out of date or inaccurate.

### Description of how this pull request fixes the issue:
- Updates the instructions
- Copy the file when building devDocs
- Link to the file from devDocs

### Known issues with pull request:
Ideally, the file would be converted to HTML, I had a quick look at doing this with sphinx but it looks like a bigger task.
There are many more considerations for how these standalone docs should be integrated with the API docs.
I'll leave that to future work dedicated to meet those goals.

### Change log entries:
None

### Code Review Checklist:

- [x] Pull Request description:
  - description is up to date
  - change log entries
- [x] Testing:
  - Unit tests
  - System (end to end) tests
  - Manual testing
- [x] API is compatible with existing add-ons.
- [x] Documentation:
  - User Documentation
  - Developer / Technical Documentation
  - Context sensitive help for GUI changes
- [x] UX of all users considered:
  - Speech 
  - Braille
  - Low Vision
  - Different web browsers
  - Localization in other languages / culture than English

### CODE REVIEW

File: nvdaHelper/readme.md [hunk 1]
```diff
@@ -11,18 +11,37 @@ You should still build on the command line to verify errors.
...
-- Other defaults are fine
-- Includes: `../include;../miscdeps/include;./;../build\x86_64;%(AdditionalIncludeDirectories)`
-- Defines: `WIN32;_WINDOWS;_USRDLL;NVDAHELPER_EXPORTS;UNICODE;_CRT_SECURE_NO_DEPRECATE;LOGLEVEL;_WIN32_WINNT;_WIN32_WINNT_WIN7;`
-- Force Includes: `winuser.h`
-- Create the project.
+  - This should have a single 'checked' item, the path to nvdaHelper
+- Other defaults are fine, press next
+- Select "use external build system" for "How do you want to build the project?", press next
+- Build command line: `scons source`
+- Include search paths: `../include;../miscdeps/include;./;../build\x86_64;%(AdditionalIncludeDirectories)`
```
Comment (lukaszgo1) [inline nvdaHelper/readme.md:22]: In the repository `miscDeps` are with a capital D. While Windows does not care about this it makes sense to be consistent IMO:
```suggestion
- Include search paths: `../include;../miscDeps/include;./;../build\x86_64;%(AdditionalIncludeDirectories)`
```

**Review by michaelDCurran — APPROVED** [review]

**Review by seanbudd — COMMENTED** [review]
No issues stand out to me, should I run through the process to confirm?

**Comment by feerrenrut** [comment]
@seanbudd yes, that would be helpful.

**Review by seanbudd — APPROVED** [review]

**Comment by seanbudd** [comment]
@feerrenrut  - followed the README, setup and confirmed the build worked properly.

**Pull request merged**